### DISCUSSION TITLE
Is the use of discussions for what would normally appear in bugs disingenuous?

### DISCUSSION THREAD

**Discussion by RebelSoftware** [livewire/livewire#6617] [4. General]
The issues tab is empty. Not because people aren't having issues, but because invariably an issue is moved to discussions where it dies an inglourious death of being ignored.

I am not saying most issues are not bugs, I am saying that the deliberate movement of issues to discussions without a response or explanation is disingenuous. I am assuming the reason is to foster the false belief that people are not having problems with Livewire or it doesn't have any shortcomings.

The number of "discussions" that go unresolved is disheartening, most of the time it is people genuinely wanting to better their understanding of Livewire and how to use it. Not everything can be gleaned from the documentation. 

For me it is making my experience a more negative one than I would like.

**Comment by joshhanley** [comment]
@RebelSoftware I’m sorry you’ve had a negative experience and we appreciate any feedback to make things better. 

I personally convert the issues to discussions and add an explanation message to each one. Did I miss some? I’ve include our reasoning for not using issues below for your reference and I can assure you it’s not because we’re trying to fill people with a false belief that Livewire has no problems.

In regard to discussions, I try to reply to almost every discussion, but have been offline for most of this week. I also try to coach people through their problems and how to submit a failing test, which makes it significantly easier for us to then fix the issue.

Ultimately we’re a small team, we’re working around the clock on Livewire and the community, as such we appreciate your patience.

Regards,
Josh

> This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by calebporzio** [comment]
Thanks for weighing in Josh.

Yeah, it's not to make Livewire seem like it has no issues, it's to prevent the list of "issues" from becoming our team's "todo list"

The reason it's a bad todo list is that: anyone can open as many issues as they want whenever they want, no matter how good or bad of an explanation of the issue it is.

There's lots more reasons, but you can read here for more: https://github.com/alpinejs/alpine/discussions/1423

**Comment by calebporzio** [comment]
Also reminder that Livewire is an open source project and the onus for helping others isn't solely on us. The ideal world is where people from the community pitch-in to help each other solve and identify problems.

Then when a problem is identified, a pull request with a failing test (at minimum) is created and I put my time towards fixing it.
